### PR TITLE
 Replace tuf roles overview image with text table

### LINKED ISSUE #66: Turn Figure 1 into a text table
Self-explanatory

### PR DESCRIPTION
**Pull request by lukpueh**
Fixes #66

### CODE REVIEW

File: pep-0458.txt [hunk 1]
```diff
@@ -337,12 +337,52 @@ paths of available target files together with their cryptographic hashes.
...
+|                 | precisely, the targets role shares the responsibility of  |
+|                 | providing information about the content of updates. The   |
+|                 | targets role may also delegate trust of repository files  |
+|                 | to other roles (delegated roles).                         |
++-----------------+-----------------------------------------------------------+
+| delegated roles | If the top-level targets role performs delegation, the    |
+|                 | resulting delegated roles can then provide their own      |
+|                 | metadata files. The format of the metadata files provided |
+|                 | by delegated targets roles is the same as that of         |
+|                 | targets.json. As with targets.json, the latest version of |
```
Comment (lukpueh) [inline pep-0458.txt:365]: Might be good to state that the targets role writes to "targets.json", at this point the reader does not know that yet (they might have a suspicion though).

File: pep-0458.txt [hunk 2]
```diff
@@ -337,12 +337,52 @@ paths of available target files together with their cryptographic hashes.
...
+|                 | metadata files belonging to delegated roles are described |
+|                 | in the snapshot role's metadata.                          |
++-----------------+-----------------------------------------------------------+
+|     snapshot    | The snapshot role is responsible for ensuring that        |
+|                 | clients see a consistent repository state. It provides    |
+|                 | repository state information by indicating the latest     |
+|                 | versions of all metadata files on the repository in       |
+|                 | snapshot.json. The only metadata file not listed in       |
+|                 | snapshot.json is timestamp.json, which is always created  |
+|                 | after snapshot.json.                                      |
```
Comment (lukpueh) [inline pep-0458.txt]: This needs to be updated to the latest version of the spec. The current spec does not have root in snapshot either.

**Review by lukpueh — COMMENTED** [review]
So far I only extracted the text from the original ` pep-0458-1.png` and put it in a text table without changing the contents. Here are two suggestion that we should fix:

**Review by trishankatdatadog — APPROVED** [review]
💯

**Comment by trishankatdatadog** [comment]
@lukpueh Go for it, do what you think is best

**Comment by lukpueh** [comment]
@trishankatdatadog, let me know what you think of https://github.com/secure-systems-lab/peps/pull/73/commits/5b8e5b1a6f18c726607b44022b795cc9aa2f213b. The targets update is not absolutely necessary. But we do need to mention that root is no longer in snapshot. Feel free to suggest alternative wording.

File: pep-0458.txt (hunk 1 above)
Comment (trishankatdatadog) [reply pep-0458.txt:365]: Go for it :yolo:

File: pep-0458.txt [hunk 3]
```diff
@@ -337,12 +337,54 @@ paths of available target files together with their cryptographic hashes.
...
+|                 | metadata files. The format of the metadata files provided |
+|                 | by delegated targets roles is the same as that of         |
+|                 | targets.json. As with targets.json, the latest version of |
+|                 | metadata files belonging to delegated roles are described |
+|                 | in the snapshot role's metadata.                          |
++-----------------+-----------------------------------------------------------+
+|     snapshot    | The snapshot role is responsible for ensuring that        |
+|                 | clients see a consistent repository state. It provides    |
+|                 | repository state information by indicating the latest     |
+|                 | versions of targets and delegated metadata files on the   |
```
Comment (trishankatdatadog) [inline pep-0458.txt]:
```suggestion
|                 | versions of the top-level targets and delegated targets metadata files on the   |
```

**Pull request merged**